doc/ref/chap-type-method.md

```diff
@@ -29,7 +29,7 @@ These two types are for OSH code only.
 A bash array holds a sequence of strings.  Some entries may be unset, i.e.
 *not* an empty string.
 
-See [sh-array][] for details.  In YSH, prefer to use [List](#List) instances.
+See [sh-array][] for the initialization.  In YSH, prefer to use [List](#List) instances.
```

Comment:
How about:

> See `sh-init-list` for initialization/mutation, and `sh-array` for other operations on BashArray.

or maybe we should called it sh-indexed-array and sh-assoc-array,  but I guess that is too long and doesn't fit

Reply:
Currently, `sh-array` only explains the special rule for the list initialization. The section doesn't explain anything about other operations. We might add the description about the operation `a[i]=v` in `sh-array`, but I'm not sure if we should describe the other operations as the filename is `chap-osh-assign`. In fact, the expansion of arrays of the form `${a[...]}` is explained in `op-bracket` in `chap-word-lang.md`.

Reply:
In commit 4c4f59333, I updated it as "See [sh-init-list][] and [sh-array][] for initialization/mutation of BashArray." for now.